runtime/src/main/java/org/corfudb/recovery/FastObjectLoader.java

```diff
@@ -518,8 +518,9 @@ private void initializeHeadAndTails() {
      */
     private void cleanUpForRetry() {
         runtime.getAddressSpaceView().invalidateClientCache();
+        // Todo(Maithem): dont expose these collections, have a clear method on the view?
```

Comment:
this todo can be addressed here, right?

Reply:
Are we going to keep this loader around ? Should we not delete this ?

Reply:
Done.

I think we should remove it in a different patch.